pkg/cmd/pulumi/pulumi.go

```diff
@@ -247,7 +247,7 @@ func NewPulumiCmd() *cobra.Command {
 
 			checkVersionMsg, ok := <-updateCheckResult
 			if ok && checkVersionMsg != nil && !isJSON {
-				cmdutil.Diag().Warningf(checkVersionMsg)
+				cmdutil.Diag().Infof(checkVersionMsg)
```

Comment:
Does `cmdutil.Diag().Warningf` output to stderr or stdout? Where does `cmdutil.Diag().Infof` output go?

If this changes the output from stderr to stdout, I'm concerned it could impact commands that output data to stdout, like `pulumi stack export`.

Reply:
I was under the impression that Warningf and Infof will both log to the same `Diag()`.

This is incorrect
https://github.com/pulumi/pulumi/blob/8c017e18cdb0382bbdd5c3733393808440fdbc85/sdk/go/common/diag/sink.go#L73-L79

`Warning uses stderr`
`Info uses stdout`

I will change this to use `Infoerr`. Thanks for the feedback!